custom/conf/app.example.ini

```diff
@@ -248,6 +248,10 @@ EVENT_SOURCE_UPDATE_TIME = 10s
 ; Whether to render SVG files as images.  If SVG rendering is disabled, SVG files are displayed as text and cannot be embedded in markdown files as images.
 ENABLE_RENDER = true
 
+[ui.csv]
+; Maximum allowed file size in bytes to render CSV files as table. (Set to 0 for no limit).
+MAX_FILE_SIZE = 524288
```

Comment:
Why did you set 512 KB as default maximum?

Reply:
I just adopted the value from Github.

> Exceeding the file size. Our rendering only works for files up to 512KB. Anything bigger than that slows down the browser. [Source](https://docs.github.com/en/github/managing-files-in-a-repository/rendering-csv-and-tsv-data)